packages/model-viewer/src/features/scene-graph/nodes/primitive-node.ts

```diff
@@ -157,7 +131,8 @@ export class PrimitiveNode extends Node {
   async enableVariant(name: string|
                       null): Promise<ThreeMaterial|ThreeMaterial[]|null> {
     if (name == null) {
-      return await this.setActiveMaterial(this[$defaultMaterialIdx]);
+      // tslint:disable-next-line:no-return-await needed by FireFox
+      return await this.setActiveMaterial(this[$initialMaterialIdx]);
```

Comment:
Thats what I thought but I tracked some flaky Firefox behavior here and this seemed to resolve it, I was thinking maybe there are circumstance where this gets treated differently on different browsers or is maybe a browser bug, or maybe I was just getting a false positive several times in a row 🤷

Reply:
When were you seeing this flakiness? I notice this change is only to add the `//tslint`, not the actual `await`. If there's a  race condition in Firefox, it could easily mean there's a race generally that just isn't  getting triggered because of perf differences between browsers. I'd like to make sure we understand the root cause.

Reply:
I added the await first, then added the comment later to prevent removing the await accidentally during the course of changes. It's easy to repro just do `npm run watch:test` then open the tests in Firefox and run.

Reply:
I just removed the await and tested again on Firefox and it had no problems, there were other synchronous issues that were addressed that probably resolved this, I think it's safe to remove.